sdk/src/main/res/values-ja/strings.xml

```diff
@@ -107,4 +108,15 @@
     <string name="payment_method_mobile_banking_bay_title">KMA</string>
     <string name="payment_method_mobile_banking_bbl_title">Bualuang mBanking</string>
     <string name="googlepay">Google Pay</string>
-</resources>
\ No newline at end of file
+    <string name="payment_method_atome_title">Atome</string>
+    <string name="label_name">Name (optional)</string>
+    <string name="label_email_optional">Email (optional)</string>
+    <string name="label_shipping">Shipping address</string>
+    <string name="label_billing">Billing address</string>
```

Comment:
this is optional

Reply:
Thanks! -> https://github.com/omise/omise-android/pull/239/commits/7e167db9d49d927cc7fe05f63b6450e7f069d377